samples/snippets/src/test/java/com/example/spanner/PgSpannerStandaloneExamplesIT.java

```diff
@@ -99,6 +100,7 @@ public static void createTestDatabase() throws Exception {
             "CREATE TABLE Venues ("
                 + "VenueId      bigint NOT NULL,"
                 + "Revenue      NUMERIC,"
+                + "VenueDetails JSONB,"
```

Comment:
This column should be added later using alter and not before

Reply:
VenueDetails column added here, is initially being dropped in `addJsonbColumn_shouldSuccessfullyAddColumn` integration test using `ALTER TABLE DROP COLUMN`, and then re-added using the sample. This is done so that all the JSONB integration tests can run independent of each other (for example, `queryWithJsonbParameter_shouldReturnResults` would not have `addJsonbColumn_shouldSuccessfullyAddColumn` as a pre-requisite in that case).

Reply:
Ohh okay. Thanks